### PR TITLE
Remove unused imports

### PR DESCRIPTION
**Pull request by Gallaecio**
https://lgtm.com/rules/6770079/

### CODE REVIEW

File: scrapy/utils/url.py [hunk 1]
```diff
@@ -12,7 +12,6 @@
 # scrapy.utils.url was moved to w3lib.url and import * ensures this
 # move doesn't break old code
 from w3lib.url import *
-from w3lib.url import _safe_chars, _unquotepath
```
Comment (kmike) [inline scrapy/utils/url.py]: I think this one was intentional

File: tox.ini [hunk 2]
```diff
@@ -73,6 +73,7 @@ commands =
 basepython = python3.8
 deps =
     {[testenv]deps}
+    -r docs/requirements.txt
```
Comment (kmike) [inline tox.ini]: is this change related to removal of unneeded imports?

Comment (Gallaecio) [reply tox.ini]: When cleaning up `pytest.ini`, I accidentally removed the `--ignore=docs/conf.py` line, which made `flake8` tests require doc dependencies. I reverted that line removal, but forgot to revert this. :man_facepalming:

**Comment by kmike** [comment]
Thanks @Gallaecio!
